lib/internal/abort_controller.js

```diff
@@ -65,10 +69,10 @@ ObjectDefineProperty(AbortSignal.prototype, SymbolToStringTag, {
 
 defineEventHandler(AbortSignal.prototype, 'abort');
 
-function createAbortSignal() {
+function createAbortSignal(aborted = false) {
```

Comment:
Why this approach rather than doing `new AbortSignal`esque?

Reply:
`new AbortSignal()` is not permitted by the spec.

Reply:
Oh, right, we ended up doing this and not the "fake parameter in constructor" approach.